src/main/java/com/faforever/client/main/MainController.java

```diff
@@ -481,6 +461,47 @@ public void onRevealLogFolder() {
     this.platformService.reveal(logPath);
   }
 
+  public class PopupDisplayer implements InvalidationListener {
```

Comment:
Move to file and rename to `ToastNotificator` or `TransientNotificator` as you like :)